### PR TITLE
Stop filtering 'other' supergroup in all-content finder

### PR DESCRIPTION
**Pull request by koetsier**
Some small changes to the all-content finder

- include the 'other' content purpose supergroup.
- relevance as the default sort order
- remove email and atom feed link

Trello: https://trello.com/c/ftI0xd67/459-make-post-review-changes-to-all-content-finder

### CODE REVIEW

File: config/finders/all_content.yml [hunk 1]
```diff
@@ -6,16 +6,12 @@ locale: en
...
-title: All content
 description: Find content from government
 signup_content_id: 49838f97-5d2e-407a-8876-30c4abe2a6bc
 details:
   default_documents_per_page: 20
   document_noun: result
   format_name: Documents
-  reject:
-    content_purpose_supergroup:
-    - other
```
Comment (bilbof) [inline config/finders/all_content.yml:18]: 😢 can we remove `reject_content_purpose_supergroup` from the email alert api if we're not using this anymore?

File: config/finders/all_content.yml [hunk 2]
```diff
@@ -6,16 +6,12 @@ locale: en
 publishing_app: rummager
 rendering_app: finder-frontend
 schema_name: finder
-title: All content
```
Comment (bilbof) [inline config/finders/all_content.yml]: Does it still work without a title? You might need to update the content schemas (either making title optional or adding a new field to specify whether the title should be displayed).

https://github.com/alphagov/govuk-content-schemas/blob/1c942c7e6f5101e69a563024630d2d27ce3de092/formats/finder.jsonnet

https://github.com/alphagov/govuk-content-schemas/blob/1c942c7e6f5101e69a563024630d2d27ce3de092/formats/shared/default_format.jsonnet#L7

Comment (koetsier) [reply config/finders/all_content.yml]: ah I did not know that. I'll add an empty field instead

Comment (koetsier) [reply config/finders/all_content.yml]: That does not work either. I put the title back for now. Removing the title is now a new card

File: config/finders/all_content.yml [hunk 3]
```diff
@@ -55,19 +51,15 @@ details:
...
-    key: "-public_timestamp"
+  - key: "-public_timestamp"
     name: Updated (newest)
   - key: public_timestamp
     name: Updated (oldest)
 routes:
 - path: "/all-content"
   type: exact
-- path: "/all-content.atom"
-  type: exact
```
Comment (bilbof) [inline config/finders/all_content.yml:71]: Will this also remove the route, or is that a separate task?

File: config/finders/all_content.yml [hunk 4]
```diff
@@ -8,14 +8,10 @@ rendering_app: finder-frontend
 schema_name: finder
 title: All content
 description: Find content from government
-signup_content_id: 49838f97-5d2e-407a-8876-30c4abe2a6bc
```
Comment (bilbof) [inline config/finders/all_content.yml:11]: Shall we also delete the email signup content item from rummager? We should unpublish it, too.

**Review by bilbof — DISMISSED** [review]

File: config/finders/all_content.yml (hunk 1 above)
Comment (koetsier) [reply config/finders/all_content.yml:18]: yes good idea

Comment (koetsier) [reply config/finders/all_content.yml:18]: Added a trello card for this

File: config/finders/all_content.yml (hunk 4 above)
Comment (koetsier) [reply config/finders/all_content.yml:11]: Yes, I'll add a point to the trello card mentioning that we should do that as part of deployment

**Review by bilbof — APPROVED** [review]

File: config/finders/all_content.yml (hunk 3 above)
Comment (koetsier) [reply config/finders/all_content.yml:71]: I suspect it will remove the route through the publishing api, but I'll make a note to check.

**Pull request merged**